microbenchmark-runner-core/src/main/java/jmh/mbr/core/ResultsWriter.java

```diff
@@ -15,28 +15,33 @@
  */
 package jmh.mbr.core;
 
-import lombok.SneakyThrows;
-
 import java.io.ByteArrayOutputStream;
 import java.io.PrintStream;
 import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
+import java.util.ServiceLoader;
 
 import org.openjdk.jmh.results.RunResult;
 import org.openjdk.jmh.results.format.ResultFormatFactory;
 import org.openjdk.jmh.results.format.ResultFormatType;
+import org.openjdk.jmh.runner.format.OutputFormat;
+
+import lombok.SneakyThrows;
 
 /**
  * @author Christoph Strobl
  */
-interface ResultsWriter {
+public interface ResultsWriter {
 
 	/**
 	 * Write the {@link RunResult}s.
+	 * @param output
 	 *
 	 * @param results can be {@literal null}.
 	 */
-	void write(Collection<RunResult> results);
+	void write(OutputFormat output, Collection<RunResult> results);
```

Comment:
What's the purpose of passing `OutputFormat` to `ResultsWriter` ? The intention of `ResultsWriter` is taking results and writing these to somewhere such as MongoDB or Elasticsearch in addition to what JMH already does.

Reply:
I want to be able to output results to the same place as JMH. Writers can ignore that parameter if they want to.